docs/rackhd/graphs.rst

```diff
@@ -31,6 +31,287 @@ Graphs are defined via a JSON definition that conform to this schema:
   - options.\<label\> (object): key, value pairs that should all be handed to a specific task
 
 
+Graph definition attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Graph Tasks**
+
+The `tasks` field in a graph definition represents the collection of tasks that make up the runtime behavior of the graph.
+The task definition is referenced by the ``injectableName`` field, and the ``label`` field is used as a reference when specifying
+dependencies for other tasks in the graph definition.
+For example, this graph will run three tasks one after the other:
+
+::
+
+    {
+        "injectableName": "Graph.Example.Linear",
+        "friendlyName": "Linear ordered tasks",
+        "tasks": [
+            {
+                "label": "task-1",
+                "injectableName": "Task.example"
```

Comment:
Shouldn't be "taskName"? "injectableName" is used in task definition, but when taskgraph references it and it will change the key to "taskName", you are review all our current task graph definition.